### PR TITLE
Use pinned cspell in Export-API.ps1 and aggregate-reports

### PR DESCRIPTION
**Pull request by danieljurek**
Supersedes https://github.com/Azure/azure-sdk-for-net/pull/26218 but without all the codeowners noise that comes from a messy rebase.

### CODE REVIEW

File: .vscode/cspell.json [hunk 1]
```diff
@@ -199,14 +200,15 @@
...
-        "Vcores",
-        "Unrestorable",
-        "Stdev",
+        "Droppeded",
+        "Nchar",
         "Ntext",
         "Nvarchar",
-        "Stdev"
+        "Stdev",
+        "Stdev",
```
Comment (AlexGhiondea) [inline .vscode/cspell.json:209]: Are these duplicates?

Comment (danieljurek) [reply .vscode/cspell.json:209]: They are... More reason to sort this list whenever adding to it.

**Review by weshaggard — APPROVED** [review]

**Comment by danieljurek** [comment]
aggregate-reports is failing on link check, unrelated to this PR

**Pull request merged**